CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Multipage Review/AddPageButtonView.swift

```diff
@@ -0,0 +1,31 @@
+//
```

Comment:
[Clarification] @vizaknaidavid  Have you tried using BottomLabelButton?

Reply:
I implemented before this was merged, but I can refactor it so I use this